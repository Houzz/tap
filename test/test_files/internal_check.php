<?php

function main() {
    $x = 1;
    __tap_check($x, "int");
    __tap_check($x, "string");
}

