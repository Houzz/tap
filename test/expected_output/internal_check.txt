6 E_INTERNAL_TYPE_CHECK
